airflow/providers/common/sql/CHANGELOG.rst

```diff
@@ -28,6 +28,14 @@ Changelog
 1.11.1
```

Comment:
1.12.0

Reply:
Fixed

Reply:
Is it braking/significant changes? We don't have to add it otherwise